crates/wallet/src/wallet/error.rs

```diff
@@ -78,8 +78,8 @@ pub enum CreateTxError {
     },
     /// When bumping a tx the absolute fee requested is lower than replaced tx absolute fee
     FeeTooLow {
-        /// Required fee absolute value (satoshi)
-        required: u64,
+        /// Required fee absolute value [`Amount`]
+        required: Amount,
```

Comment:
I think we should fix the displayed error message for `FeeTooLow`, maybe by using `display_dynamic` again

Reply:
Yes, updated.
I did another pass on the codebase as well and currently didn't find other spots that would need this kind of fix.